DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/ArcGISProjectUtils.cs

```diff
@@ -0,0 +1,66 @@
+using ArcGIS.Core.Data;
+using ArcGIS.Core.Data.DDL;
+using ArcGIS.Desktop.Core;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+
+namespace Speckle.Converters.ArcGIS3.Utils;
+
+public class ArcGISProjectUtils
+{
+  public string GetDatabasePath()
+  {
+    string fGdbName = "Speckle.gdb";
+    string fGdbPath;
+    try
+    {
+      var parentDirectory = Directory.GetParent(Project.Current.URI);
+      if (parentDirectory == null)
+      {
+        throw new ArgumentException($"Project directory {Project.Current.URI} not found");
+      }
+      fGdbPath = parentDirectory.ToString();
+    }
+    catch (Exception)
```

Comment:
Eek! Let's be specifc here please

Reply:
the list of possible exceptions is quite long, not sure how to handle this: 
![image](https://github.com/specklesystems/speckle-sharp/assets/89912278/9346cf19-3089-4415-99dc-754f0589b56e)